service/src/test/java/org/cbioportal/service/util/AlterationEnrichmentUtilTest.java

```diff
@@ -120,7 +121,7 @@ public void createAlterationEnrichments() throws Exception {
         AlterationEnrichment alterationEnrichment2 = result.get(1);
         Assert.assertEquals((Integer) 3, alterationEnrichment2.getEntrezGeneId());
         Assert.assertEquals("HUGO3", alterationEnrichment2.getHugoGeneSymbol());
-        Assert.assertEquals("CYTOBAND3", alterationEnrichment2.getCytoband());
+        Assert.assertEquals(null, alterationEnrichment2.getCytoband());
```

Comment:
this seems incorrect?

Reply:
Hi @inodb , that is right. for alterations we will have to fill out cytobands in the frontend, since alterations have no reference genome information associated with them. We will have to rely on the reference genome information of the study to backfill cytobands in the frontend.